Jenkinsfile

```diff
@@ -153,7 +153,7 @@ pipeline {
                         source ${PWD}/.envrc
                         set -x
                         unset HCF_PACKAGE_COMPILATION_CACHE
-                        rm -f scf-*-amd64-*.zip
+                        rm -f scf-*amd64*.zip
```

Comment:
It _would_ be more specific, which is why I didn't choose it… I aimed for something that would match both the old name as well as the new one, in case we decide to change things around _again_.